### ISSUE TITLE
Crash on clicking "Add from image" button in scan activity

### ISSUE THREAD

**Issue by TheLastProject** [CatimaLoyalty/Android#856]
A device lacking a file browser it seems. Can probably look at the import/export activity and see how it deals with a missing file manager there.

This crash is rare (only affects users with very cursed devices or configurations), but it's still a crash and an easy one to fix.

```
android.content.ActivityNotFoundException: 
  at android.app.Instrumentation.checkStartActivityResult (Instrumentation.java:1937)
  at android.app.Instrumentation.execStartActivity (Instrumentation.java:1616)
  at android.app.Activity.startActivityForResult (Activity.java:4501)
  at androidx.activity.ComponentActivity.startActivityForResult (ComponentActivity.java:597)
  at androidx.core.app.ActivityCompat.startActivityForResult (ActivityCompat.java:239)
  at androidx.activity.ComponentActivity$2.onLaunch (ComponentActivity.java:210)
  at androidx.activity.result.ActivityResultRegistry$2.launch (ActivityResultRegistry.java:167)
  at androidx.activity.result.ActivityResultLauncher.launch (ActivityResultLauncher.java:47)
  at protect.card_locker.ScanActivity.addFromImage (ScanActivity.java:209)
  at protect.card_locker.ScanActivity$$ExternalSyntheticLambda0.onClick (Unknown Source:2)
  at android.view.View.performClick (View.java:6308)
  at com.google.android.material.button.MaterialButton.performClick (MaterialButton.java:1131)
  at android.view.View$PerformClick.run (View.java:24949)
  at android.os.Handler.handleCallback (Handler.java:790)
  at android.os.Handler.dispatchMessage (Handler.java:99)
  at android.os.Looper.loop (Looper.java:164)
  at android.app.ActivityThread.main (ActivityThread.java:6569)
  at java.lang.reflect.Method.invoke (Native Method)
  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:438)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:857)
```

**Comment by Altonss** [comment]
Maybe this one could be closed thanks to #920 ?

**Comment by TheLastProject** [comment]
Thanks for always paying such close attention @Altonss :)
